scripts/examples/gn_efr32_example.sh

```diff
@@ -169,6 +171,7 @@ else
                     exit 1
                 fi
                 USE_WIFI=true
+                optArgs+="chip_device_platform =\"efr32\" "
```

Comment:
You do it later on if the other ifs line 255 and 272. Same for the `USE_WIFI=true`

You can delete this one

Reply:
Nope we need to keep this one. 
This one is for the build option `--wifi 9116`
Line 255 if for the build argument `use_rs9116=true`
And last one is for the 917. 

i know it's a lot of duplication but they are all needed to support the same behaviour that we once had.